### PR TITLE
Visual updates to pledge screen

### PR DESCRIPTION
**Pull request by cdolm92**
# 📲 What

Updating the pledge screen so that visual experience is consistent to users.

# 👀 See

Snapshots below. Screenshots coming soon 

| Before 🐛 | After 🦋 |
| --- | --- |
| IMG_3817| Simulator Screen Shot - iPhone Xs - 2019-08-13 at 11 40 22 
| --- | --- |
|IMG_3825|Simulator Screen Shot - iPhone Xs - 2019-08-13 at 16 33 09|

# ✅ Acceptance criteria

- [ ]  Compare UI to designs on Abstract

# ⏰ TODO

- [x] self-review

### CODE REVIEW

**Comment by jamielynnroth** [comment]
Based on screenshots, this looks good (comparing to https://share.goabstract.com/f67d7fa5-50ba-4999-b6fe-f6a682647cf7?collectionLayerId=4070503d-c373-4975-b9cc-7622093aca82&mode=design) 

For the ACs not marked below, will wait to test them in the build. 

ACs: 
- Rounding of corners has been set to 18pt ✅
- Reward detail font style changes have been applied ✅
- Reward detail font color changes have been applied ✅
- Background border/stroke visual tweak has been applied ✅
- Green ✔️on left-hand side has been removed
- The darker-grey background has been removed (should be all white inside) ✅
- Visual styling of pledge and shipping fields has been tweaked ✅
- The`...more` has been changed to be ksr_green_500
- Accountability paragraph text styling tweaks have been applied  ✅
- All updates match screens in Abstract for the back this project flow 
- All updates match screens in Abstract for the manage pledge flow

File: Kickstarter-iOS/Views/Storyboards/RewardPledge.storyboard [hunk 1]
```diff
@@ -21,39 +21,30 @@
...
                                         <subviews>
-                                            <view contentMode="scaleToFill" misplaced="YES" translatesAutoresizingMaskIntoConstraints="NO" id="C2f-Es-FaN">
-                                                <rect key="frame" x="0.0" y="0.0" width="384" height="1190"/>
-                                                <subviews>
-                                                    <view contentMode="scaleToFill" misplaced="YES" translatesAutoresizingMaskIntoConstraints="NO" id="fJb-oy-BtY">
-                                                        <rect key="frame" x="-8" y="224" width="400" height="966"/>
-                                                        <color key="backgroundColor" red="0.89916485548019409" green="0.89916485548019409" blue="0.89916485548019409" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-                                                    </view>
-                                                </subviews>
+                                            <view contentMode="scaleToFill" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="C2f-Es-FaN">
```
Comment (ifbarrera) [inline Kickstarter-iOS/Views/Storyboards/RewardPledge.storyboard]: Hmm looks like there's a few views with ambiguous sizes/positions in the storyboard.

Comment (ifbarrera) [reply Kickstarter-iOS/Views/Storyboards/RewardPledge.storyboard]: @cdolm92 were you able to take a look at these ambiguous/conflicting constraints on the storyboard?

File: Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewController.swift [hunk 2]
```diff
@@ -621,8 +581,8 @@ internal final class DeprecatedRewardPledgeViewController: UIViewController {
 
     for (idx, item) in allItems.enumerated() {
       let label = UILabel()
-        |> UILabel.lens.font .~ (idx == 0 ? .ksr_headline(size: 13) : .ksr_body(size: 14))
-        |> UILabel.lens.textColor .~ (idx == 0 ? .ksr_soft_black : .ksr_text_dark_grey_500)
+        |> UILabel.lens.font .~ (idx == 0 ? UIFont.ksr_caption1(size: 16).bolded : .ksr_body())
```
Comment (ifbarrera) [inline Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewController.swift]: I think for the `Includes` title label the font should be the same as `Description` and `Estimated Delivery` which are both `UIFont.ksr_callout().bolded`.

**Comment by ifbarrera** [comment]
✅ Tested on:

iPhone 6s (11.4)
iPhone 5s (12.4)
iPhone X (12.4)
iPad Pro 11 inch (12.4)

**Pull request merged**